.github/workflows/tests.yaml

```diff
@@ -13,18 +13,31 @@ jobs:
       matrix:
         platform: [ubuntu-latest, windows-latest]
         php: [ "7.4", "8.0", "8.1", "8.2" ]
-        extensions: ["grpc-1.49.0"]
+        extensions: ["grpc-1.54.0"]
```

Comment:
If we dont specify the version, it should automatically pick the latest version, right?

Reply:
It's a very good question - the reason I specified it here is because it _wasn't_ picking up the latest version. It seems to be picking up `grpc-1.41.1`, and I don't know why. This is also fine, because that version works. So I think it's possible somebody pinned the working version because of the same issue we were having.

I did a little bit of digging and still couldn't find the source, so I just updated it here instead (and left the emulators on just `grpc`, because as long as our main tests pass on it, I think that's sufficient). I would be happy to remove this at some point though, because I agree that the way it is isn't easy to maintain :-/